statsmodels/graphics/tests/test_functional.py

```diff
@@ -116,36 +121,37 @@ def test_hdr_bw(close_figures):
     median_t = [24.25, 25.64, 25.99, 25.04, 23.71, 22.38,
                 21.31, 20.44, 20.24, 20.51, 21.19, 22.38]
     assert_almost_equal(hdr.median, median_t, decimal=2)
+    assert_equal(data, orig_data)
 
 
 @pytest.mark.skipif(not have_matplotlib, reason='matplotlib not available')
 def test_hdr_ncomp(close_figures):
     fig, hdr = hdrboxplot(data, ncomp=3)
     median_t = [24.33, 25.71, 26.04, 25.08, 23.74, 22.40,
-                21.32, 20.45, 20.25, 20.53, 21.2 , 22.39]
+                21.32, 20.45, 20.25, 20.53, 21.2, 22.39]
     assert_almost_equal(hdr.median, median_t, decimal=2)
+    assert_equal(data, orig_data)
 
 
 def test_banddepth_BD2():
     xx = np.arange(500) / 150.
     y1 = 1 + 0.5 * np.sin(xx)
-    y2 = 0.3 + np.sin(xx + np.pi/6)
-    y3 = -0.5 + np.sin(xx + np.pi/6)
-    y4 = -1 + 0.3 * np.cos(xx + np.pi/6)
+    y2 = 0.3 + np.sin(xx + np.pi / 6)
+    y3 = -0.5 + np.sin(xx + np.pi / 6)
+    y4 = -1 + 0.3 * np.cos(xx + np.pi / 6)
 
     data = np.asarray([y1, y2, y3, y4])
     depth = banddepth(data, method='BD2')
-    expected_depth = [0.5, 5./6, 5./6, 0.5]
+    expected_depth = [0.5, 5. / 6, 5. / 6, 0.5]
     assert_almost_equal(depth, expected_depth)
-
     ## Plot to visualize why we expect this output
-    #fig = plt.figure()
-    #ax = fig.add_subplot(111)
-    #for ii, yy in enumerate([y1, y2, y3, y4]):
+    # fig = plt.figure()
+    # ax = fig.add_subplot(111)
+    # for ii, yy in enumerate([y1, y2, y3, y4]):
```

Comment:
Are these worth keeping as comments?  Or actually just defunct code?  If it's the latter then second-best (after deleting) is to keep them pep8-invalid so they don't fall through the cracks.